.ci/commits-check.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+echo "** Running commits message check..."
+exit_code=0
+readarray -t commits <<< $(git log --no-merges -n "${COMMIT_COUNT}" --format='%H')
+for commit in "${commits[@]}"; do
+  AUTHOR=$(git log -1 --format='%aN <%aE>' "${commit}")
+  HEADER=$(git log -1 --format='%s' "${commit}")
+  # Check header length
+  if [ $(echo "${HEADER}" | wc -L) -gt 72 ]; then
+    echo "The commit header of ${commit} is longer than 72 characters"
+    echo "Header is: ${HEADER}"
+    exit_code=1
+  fi
+  # Check if commit header ends with '.'
```

Comment:
Hi @PaulYuuu , do we need to check the blank spaces in the end? Actually, I mean should the last char of the header be [0-9,a-z,A-Z] only?

Reply:
I think not only `[0-9,a-z,A-Z]`, we should also consider ```[`"']```, like `"str"`, ``` `code block` ```. Let me update the rule to check `.` and ` `, for other chars, let's improve the rule case by case in the future.